settings.js

```diff
@@ -120,7 +120,7 @@ window.spinnakerSettings = {
   authEnabled: authEnabled,
   authTtl: 600000,
   gitSources: ['stash', 'github', 'bitbucket'],
-  triggerTypes: ['git', 'pipeline', 'docker', 'cron', 'jenkins', 'travis'],
+  triggerTypes: ['git', 'pipeline', 'docker', 'cron', 'jenkins', 'travis', 'pubsub'],
```

Comment:
Thoughts about whether this should be enabled by default at this point? Maybe the `pubsub` entry can wait until this is more thoroughly supported?